kart/commit.py

```diff
@@ -35,13 +37,55 @@
 from .repo import KartRepoFiles
 
 
+class CommitDiffWriter(BaseDiffWriter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if not self.spatial_filter.match_all:
+            self.RECORD_SPATIAL_FILTER_STATS = True
```

Comment:
this thing seems like it shouldn't be styled like a constant

Reply:
Done